### DISCUSSION TITLE
games on sd card

### DISCUSSION THREAD

**Discussion by e1milio** [aldostools/webMAN-MOD#1012] [Ideas]
Could the sd card reader be used to load games? It would be good to give the reader some use and expand the storage capacity of the console.

**Comment by aldostools** [comment]
SD card (SD),  Memory Stick (MS) and CompactFlash (CF) card formatted in FAT32 are supported only PS3 phat models. 

USB storage devices formatted in FAT32, exFAT, NTFS and ext2/3/4 are also supported. 

Take in count that PS2 games in ISO or encrypted BIN.ENC formats are only supported from the internal HDD due a limitation of the PS2 emulator and Cobra payload.
